### ISSUE TITLE
Remove dialog-normalization from streams (AC3)

### ISSUE THREAD

**Issue by hubblec4** [mbunkus/mkvtoolnix#1981]
https://forum.doom9.org/showthread.php?p=1806363#post1806363

For direct muxing Bluray-content (m2ts or mpls).

**Comment by remuxer32** [comment]
It's a great idea. I like and fully support it. Especially that I've been working with many Blu-rays for many years, and such a feature will be useful for me, too.

**Comment by remuxer32** [comment]
**Important information**: dialog normalization does not only apply to AC-3, but it's also used for audio codecs such as DTS & Dolby TrueHD. The only exception is LPCM for which it doesn't apply at all. I've just found a long list of BD titles (not complete because it comes from 2009) where you can see which movies (titles) have dialog normalization for which codecs. Here is a direct link: 

- http://forum.blu-ray.com/showpost.php?s=4f3be9a6733b0392d2ce88813dec64f2&p=2505236&postcount=2

My question is: would it be possible to implement removing dialog normalization not only for AC-3, but for other codecs as well? That would be definitely awesome and useful. Thank you.

**Comment by hubblec4** [comment]
Yes all other codecs should be supported also if is possible. So I asked for "streams".

**Comment by mbunkus** [comment]
At the moment I don't even know yet how that removal would be done technically. Therefore I cannot say how easy it would be to implement.

**Comment by mkver** [comment]
For ac3, there are five bits in the header of every ac3 packet that need to be changed (set to 00001, because 00000 is reserved); see section 5.3.2 of [the standard](http://atsc.org/wp-content/uploads/2015/03/A52-201212-17.pdf). But beware: There are two more CRC16 fields that need to be updated, too, or decoders might conclude that the packets are defect.

**Comment by mbunkus** [comment]
This project has been moved to [GitLab](https://gitlab.com/mbunkus/mkvtoolnix/). The corresponding issue is [here](https://gitlab.com/mbunkus/mkvtoolnix/issues/1981).
